### PR TITLE
Added Initial OpenAPI definition

### PR DESCRIPTION
**Pull request by chrisnappin**
# MES-768
* Created OpenAPI definition for the Initial Journal Only release REST API operations.
* Added speccy, run ```npm run check``` to lint the definition. Also manually tested the file in the Swagger editor (https://editor.swagger.io)
* Since Swagger/OpenAPI doesn't support recent JSON Schema, and speccy.io doesn't seem to support converting complex JSON schema to OpenAPI schema format (defect raised with them, see https://github.com/wework/speccy/issues/308), any complex types have been replaced by a _Dummy_ schema object

## Pull Request checklist

- [*] [WIP] tag removed from PR title
- [*] PR has an understandable description

## Git feature branch checklist

- [*] branch name comply with our branching strategy
- [*] git branch contains relevant JIRA ticket number
- [*] branch rebased against the latest develop

## Sign off process checklist

- [*] Code has been tested manually
- [ ] Tested by QA
- [ ] PO's approval
- [ ] Reviewers selected in Github
- [ ] PR link added to JIRA

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+openapi: 3.0.2
+info:
+  title: DVSA Mobile Examiner Services REST API
+  description: >
+    The REST API for the DVSA Mobile Examiner Services (MES) system, supporting the iPad app used to
+    record practical driving tests. Each API call needs to include a valid JWT token obtained from
+    DVSA Azure Active Directory, for a staff id recorded as an active examiner within the TARS system.
+      
+    Since Open API v3 doesn't support recent JSON Schema, any requests or responses with comnplex JSON
```
Comment (johnmorleybjss) [inline openapi.yaml]: Typo here.

**Comment by ammarhaiderbjss** [comment]
@chrisnappin What do you think about grouping these definitions under their own directories, much like the schemas? Once we add the definitions for all the other services, it might get unwieldy if its all housed in the root.

File: openapi.yaml (hunk 1 above)
Comment (chrisnappin) [reply openapi.yaml]: Great, thanks!

**Comment by chrisnappin** [comment]
Yes, we could restructure when it gets bigger. Speccy.io also says it supports combining files together (via ```$ref``` expressions) which might be a nice way to work.

**Pull request merged**